k8s/k8s.go

```diff
@@ -336,17 +322,16 @@ func CmdDelK8s(ctx context.Context, c calicoclient.Interface, epIDs utils.WEPIde
 			logger.WithField("WorkloadEndpoint", epIDs.WEPName).Warning("WorkloadEndpoint does not exist in the datastore, moving forward with the clean up")
 		} else {
 			// Could not connect to datastore (connection refused, unauthorized, etc.)
-			// so we have no way of knowing/checking ActiveInstanceID. To protect the endpoint
+			// so we have no way of knowing/checking ContainerID. To protect the endpoint
 			// from false DEL, we return the error without deleting/cleaning up.
 			return err
 		}
 
-		// Check if ActiveInstanceID is populated (it will be an empty string "" if it was populated
+		// Check if ContainerID is populated (it will be an empty string "" if it was populated
 		// before this field was added to the API), and if it is there then compare it with ContainerID
 		// passed by the orchestrator to make sure they are the same, return without deleting if they aren't.
 	} else if wep.Spec.ContainerID != "" && args.ContainerID != wep.Spec.ContainerID {
-		logger.WithField("WorkloadEndpoint", wep).Warning("CNI_CONTAINERID does not match WorkloadEndpoint ActiveInstanceID so ignoring the DELETE cmd.")
-		return nil
+		logger.WithField("WorkloadEndpoint", wep).Warning("CNI_CONTAINERID does not match WorkloadEndpoint ConainerID, don't delete WEP.")
```

Comment:
shouldn't we return after this log? (according to the comment above)

Reply:
Nope, the comment is out of date now. We used to return, but now we let execution continue below so we can clean up the IPAM info.